### PR TITLE
Added 07/20 Upcoming Releases Update

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/pages/preview/upcoming-releases.mdx [hunk 1]
```diff
@@ -23,7 +23,8 @@ Application Access.
 Teleport 10.2 will bring file upload/download support to Desktop Access,
 host certificate support to Machine ID, Elasticsearch support to Database Access,
 automatic configuration of AD to Desktop Access, and remove the need for a PVC
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:25]:
```suggestion
and automatic configuration of AD to Desktop Access. Teleport 10.2 will remove the need for a PVC
```
To break up this sentence a bit and preserve the parallel structure of the first list, which makes it slightly more readable.

File: docs/pages/preview/upcoming-releases.mdx [hunk 2]
```diff
@@ -23,7 +23,8 @@ Application Access.
...
-to securely deploy Teleport within a Kubernetes cluster.
+to securely deploy Teleport within a Kubernetes cluster, and automatic discovery
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:26]:
```suggestion
to securely deploy Teleport within a Kubernetes cluster, and will add automatic discovery
```

File: docs/pages/preview/upcoming-releases.mdx [hunk 3]
```diff
@@ -42,20 +43,14 @@ Teleport 10 will bring Cassandra support to Database Access.
 
 ## Teleport Cloud
 
-The key areas of focus for Teleport Cloud in Summer 2022 are:
+The key deliverables for Teleport Cloud in the next quarter.
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:46]:
```suggestion
The key deliverables for Teleport Cloud in the next quarter:
```

File: docs/pages/preview/upcoming-releases.mdx [hunk 4]
```diff
@@ -42,20 +43,14 @@ Teleport 10 will bring Cassandra support to Database Access.
...
-
-| Date           | Description |
-|----------------|-------------|
-| September 2022 | Complete rollout of our [worldwide access design](https://goteleport.com/blog/access-proxy-latency/), allowing low-latency access from around the world. |
-| Q2 2022        | Update security audit from independent auditors. [2021 Audit](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/)  |
-| Q2 2022        | Provide a public listing of IP addresses used by Teleport infrastructure for firewall configuration |
+| Date    | Description |
+|---------|-------------|
+| Q2 2022 | Update security audit from independent auditors. [2021 Audit](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/)  |
+| Q2 2022 | Provide a public listing of IP addresses used by Teleport infrastructure for firewall configuration |
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:51]:
```suggestion
| Q2 2022 | Provide a public listing of IP addresses used by Teleport infrastructure for firewall configuration. |
```

File: docs/pages/preview/upcoming-releases.mdx [hunk 5]
```diff
@@ -42,20 +43,14 @@ Teleport 10 will bring Cassandra support to Database Access.
...
+| Q2 2022 | Scheduled upgrades will limit downtime to non-peak hours
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:52]:
```suggestion
| Q2 2022 | Scheduled upgrades will limit downtime to non-peak hours.
```

**Review by ptgott — APPROVED** [review]

**Pull request merged**